utils/nfc-mfclassic.c

```diff
@@ -587,6 +587,7 @@ print_usage(const char *pcProgramName)
   printf("                  *** unlocked read does not require authentication and will reveal A and B keys\n");
   printf("                  *** note that unlocked write will attempt to overwrite block 0 including UID\n");
   printf("                  *** unlocking only works with special Mifare 1K cards (Chinese clones)\n");
+  printf("                  *** unlocking (w) also works for rewriting block 0 in gen2 clones\n");
```

Comment:
should be (W) ?

Reply:
Yes. With a uppercase W, we set the write_block_zero to true, that goes to unlock_card(), that try to sends unlock command for a gen1 tag, if the first unlock command fails, set magic2 to true so in the end sets writing to block 0 with auth. If unlocks commands don't fails its goes to normal gen1 behaviour so writing all blocks including 0 without auth. With a lowercase w all goes like before. All the reading process behaviour haven't changed (at least with that read-write check in unlock_card()).